contrib/ci/run

```diff
@@ -17,6 +17,11 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+# USER does not have to be define in containers
```

Comment:
I see podman is being called with
~~~
  sudo podman exec                                 \
    --user "$user"                                 \
    --env "USER=$user"                             \
~~~

Therefore I wonder if we should instead return an error when USER is unset, to avoid overriding the USER variable.

Reply:
USER is populated from su/login, it is not available in containers. I added it to podman exec to avoid possible future problems when USER is expected, but in my opinion it is good not to rely on it in case somebody runs it locally.

Reply:
Okay no problem, can we just echo a message that USER is not set?

Reply:
Done.